langwatch/src/components/GraphsLayout.tsx

```diff
@@ -1,10 +1,313 @@
-import { Container, HStack, Text, VStack } from "@chakra-ui/react";
-import type { PropsWithChildren } from "react";
+import {
+  Box,
+  Container,
+  HStack,
+  Input,
+  Spinner,
+  VStack,
+} from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import { type PropsWithChildren, useEffect, useRef, useState } from "react";
+import { ArrowDown, ArrowUp, Edit2, MoreVertical, Plus, Trash2 } from "lucide-react";
 import { DashboardLayout } from "~/components/DashboardLayout";
 import { MenuLink } from "~/components/MenuLink";
 import { useOrganizationTeamProject } from "~/hooks/useOrganizationTeamProject";
+import { api } from "~/utils/api";
```

Comment:
Layouts probably shouldn't be tied to the API

Reply:
this is not the api in the backend sense, it's the trpc endpoint, but yeah its being used by `CustomDashboardsSection` which don't belong here, it should be extracted